pkg/Library/Mediapipe/MediapipeNodes.js

```diff
@@ -81,11 +81,15 @@ export const HolisticSticker = {
       $type: 'Image',
       noinspect: true,
       nomonitor: true
+    },
+    sticker: {
+      $type: "Image",
```

Comment:
Single ' instead of " ... I don't make the rules. (Ok, I kinda do.)

Reply:
Done!